.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,14 @@
+blank_issues_enabled: false
+contact_links:
+  - name: GitHub Discussions
+    url: https://github.com/mermaid-js/mermaid/discussions
+    about: Ask the Community questions or share your own graphs in our discussions.
+  - name: Slack
+    url: https://join.slack.com/t/mermaid-talk/shared_invite/enQtNzc4NDIyNzk4OTAyLWVhYjQxOTI2OTg4YmE1ZmJkY2Y4MTU3ODliYmIwOTY3NDJlYjA0YjIyZTdkMDMyZTUwOGI0NjEzYmEwODcwOTE
+    about: Join our Community on Slack for Help and a casual chat.
+  - name: Documentation
+    url: https://mermaid-js.github.io
+    about: Read our documentation for all that Mermaid.js can offer.
+  - name: Live Editor
+    url: https://mermaid.live
+    about: Try the live editor to preview graphs in no time.
```

Comment:
Something to maybe think about in a future PR, but maybe we should add the security email from https://github.com/mermaid-js/mermaid#reporting-vulnerabilities

Maybe we should also add that information to a `SECURITY.md` file, since that's GitHub recommended way of doing things: https://docs.github.com/en/code-security/getting-started/adding-a-security-policy-to-your-repository

Reply:
> Maybe we should also add that information to a `SECURITY.md` file, since that's GitHub recommended way of doing things: https://docs.github.com/en/code-security/getting-started/adding-a-security-policy-to-your-repository

If such a file is added should be perhaps a org-level file, so that it is available in all repositories without the need of copy-pasting the same file X times.
That's of course only if you want the same file covering all repos...

Reply:
> If such a file is added should be perhaps a org-level file, so that it is available in all repositories without the need of copy-pasting the same file X times.

:+1: That makes a lot of sense, I didn't even know that feature existed (docs here: https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/creating-a-default-community-health-file)

> Not sure if emails would work here well, since afaik are all fields required and one is url, so a link is needed there...
> I'm not sure if `mailto:email` would work in that option...

True. Maybe that's another reason to add a `SECURITY.md` file, so we can link to it from here!

Reply:
Added https://github.com/mermaid-js/.github/blob/main/SECURITY.md